_posts/2022-11-02-gsoc2022-final.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "GSoC / Outreachy 2022 wrap-up"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
+title="Google Summer of Code / Outreachy 2022" alt="Google Summer of Code / Outreachy 2022"
+style="float: right; height: 12em; " />
+</p>
+
+This year MDAnalysis participated in two student programs with wonderful students from around the globe. 
+We continued our participation in [Google Summer of Code][gsoc] for another year and also participated in [Outreachy][outreachy]! for the first time. 
```

Comment:
Do we need to mention NumFOCUS here — did they pay for Outreachy?

Reply:
They are like a low-tier sponsor of outreachy, I'm not sure we need to mention them  in particular?

Reply:
I think CZI paid for our outreachy participation iirc